script.js

```diff
@@ -1,4 +1,32 @@
+var winCount = 0;
+var lostCount = 0;
+var drawCount = 0;
+var count = 0;
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  ++count;
+  if ((input == 'scissors') || (input == 'stone') || (input == 'paper')) {
+    var myOutputValue = getResult(input);
+    return myOutputValue;
+  };
+  return "The input options are scissors, stone or paper. Kindly select from these";
```

Comment:
I like how you have a default case, saves the lines of code you would have had to write.